clients/fides-consent/package.json

```diff
@@ -18,7 +18,8 @@
     "url": "https://github.com/ethyca/fides.git",
     "directory": "clients/privacy-center/packages/fides-consent"
   },
-  "module": "./dist/fides-consent.mjs",
+  "main": "dist/fides-consent.mjs",
+  "module": "dist/fides-consent.mjs",
```

Comment:
Warning: I'm going to get really annoying and ask to do a lot of renaming here... (e.g. fides-consent -> fides-components) but let's stabilize first 😄

Reply:
yeahhh I was unsure whether to tackle that renaming here or in another PR- but I suppose we could do here after we're at a stable place